### DISCUSSION TITLE
👋 Introduce Yourself

### DISCUSSION THREAD

**Discussion by blefnk** [blefnk/relivator#2] [Announcements]
## 👋 Welcome!

We’re using discussions as a place to connect with other members of our community. We hope that you:

* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by blefnk** [comment]
### 👋 Hello, everyone!

Thanks so much for everyone's interest in the Relivator! Let me shortly introduce myself 👋 My name is Nazar**ii** Korn**ii**enko, and informally just Nazar Kornienko. I'm from Ukraine 🇺🇦, and currently studying Master's Degree as Web Design in Poland 🇵🇱. My passion and day job both involve web dev and UX design.

### 🙌 Let's Be Friends!

I enjoy collaborating with others and learning new things. I believe that every interaction is a chance to learn something new, be it a design technique or a fascinating fact about someone else's experiences. 

I'm on here hoping to connect with people who are as curious and enthusiastic about life as I am. If you love a good conversation about everything from design or code to what makes you tick, let's chat!

### 🙂 Follow Me Everywhere

[GitHub](https://github.com/blefnk) | [Discord](https://discord.gg/Pb8uKbwpsJ) | [Twitter/𝕏](https://x.com/blefnk) | [Fiverr](https://fiverr.com/blefnk) | [LinkedIn](https://linkedin.com/in/blefnk) | [Facebook](https://facebook.com/blefnk)

**Comment by bostonbottleseu** [comment]
Hi @blefnk , I appreciate this repo so much, just - could i get a view into the demo, as the demo website url seems not to be working.
Also, do you think this template can be used for single product selling?

**Comment by blefnk** [comment]
@bostonbottleseu Hello and welcome to the community! Thank you for your interest in the project.

Unfortunately, the demo is currently unavailable as PlanetScale has canceled the free plan it was connected to. However, I'm excited to announce that the v1.2.6 release is coming on August 1st. This new version will feature many enhanced functionalities.

By the way, the new version uses Neon+Postgres instead of PlanetScale+MySQL. If you need another provider, Relivator's flexibility allows you to switch between many features easily. You can learn more about v1.2.6 on issues pages #38 and #32.

Regarding single product selling, the current template state is designed for 'multiple stores with multiple products.' v1.2.6 continues this approach. However, v1.3.0 will introduce new options, including single product stores.

Please stay tuned for updates and [join project's Discord](https://discord.gg/Pb8uKbwpsJ) for the latest project news. Once again, welcome to the community!
